validator/src/commands/repair_whitelist/mod.rs

```diff
@@ -52,16 +86,17 @@ pub fn command(_default_args: &DefaultArgs) -> App<'_, '_> {
 pub fn execute(matches: &ArgMatches, ledger_path: &Path) {
     match matches.subcommand() {
         ("get", Some(subcommand_matches)) => {
-            let output_mode = subcommand_matches.value_of("output");
+            let args = RepairWhitelistGetArgs::from_clap_arg_match(subcommand_matches);
```

Comment:
nit: In all your other PR's, you use the "longer" name. Ie
```
let repair_whitelist_get_args = ...
```
I actually don't mind use of the shorter name of `args` here and in those other PR's. However, I do like consistency so if you want to stick with shorter names, I think it'd be nice to do so in all of the PR's